### PR TITLE
temp: `x` script for quickly running & seeing the diff

### PR DESCRIPTION
**Pull request by kiprasmel**
the run.js is cool, but it became annoying to create source files & undo the changes, so this just helps.

`x` probably shouldn't be tracked here; i myself move it into a script directory that's inside my $PATH, e.g. `~/.local/bin/`.

i edit the content directly there, most often at the bottom because that's the latest stuff you see in the terminal when you run it.

running is the same as run.js - `x flow cui5 path/to/dir/or/file/or/whatever`

### CODE REVIEW

**Comment by danieldelcore** [comment]
Great work on all of this! Sorry I've been MIA lately. I'll have a look at this tomorrow hopefully! Keep it up

**Comment by kiprasmel** [comment]
yeah no worries. i think i've realized that this is nice for convenience for small things, but when you start adding more stuff, you're likely to miss it unless you also have tests - something to keep in mind.

**Comment by danieldelcore** [comment]
Hey @kiprasmel 👋 How's it going? 

I just noticed that you've done loads of work in your fork to setup this codeshift package: 
https://github.com/pipedrive/CodeshiftCommunity/tree/temp/quick-run-script/community/%40pipedrive__convention-ui-react

Since it's quite bespoke and has its own build configuration (jest, ts, etc) have you thought about creating this as a standalone package? This means you wont need to fork and maintain the rest of the repository and just focus the codemod package [that you've setup here](https://github.com/pipedrive/CodeshiftCommunity/tree/temp/quick-run-script/community/%40pipedrive__convention-ui-react). 

This guide should show you how it's done: 
https://www.codeshiftcommunity.com/docs/external-packages

Also the authoring guides explain the different publishing options and why you might want to use one over the other:
https://www.codeshiftcommunity.com/docs/authoring

In short 
- `npx @codeshift/cli init @pipedrive/convention-ui-react`
- Copy source files from into the new project
- `npm build` && `npm publish`


Then your codemods should be runnable like so: 

`npx @codeshift/cli -p @pipedrive/convention-ui-react@5.0.0 path/to/source`

The community folder is designed to be a bit restrictive so additional dependencies and build configuration to add complexity to the community etc.

**Comment by kiprasmel** [comment]
hey @danieldelcore, thanks! how are you?

this branch specifically was just to showcase the `x` script which i thought could've been to interest for you. i think i forgot to rebase and that's why there's a lot of unrelated stuff here.

the proper branch of ours is the `fork` one, it should be way more up to date, if you were curious about exploring our solution:
https://github.com/pipedrive/CodeshiftCommunity/tree/fork/community/%40pipedrive__convention-ui-react

as for the community folder vs standalone package, i saw that you introduced this capability a little while ago.
we actually wanted to migrate to a standalone package, but we cannot until https://github.com/CodeshiftCommunity/CodeshiftCommunity/pull/97 is merged -- would it be possible to get it moving?

thanks!
